projects/storefrontlib/src/cms-components/checkout/components/payment-type/payment-type.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="row">
+  <div class="col-md-12 col-lg-9">
+    <h3 class="cx-checkout-title d-none d-lg-block d-xl-block">
+      {{ 'checkoutProgress.paymentType' | cxTranslate }}
+    </h3>
+
+    <ng-container *ngIf="(paymentTypes$ | async)?.length; else loading">
+      <div class="form-check" *ngFor="let type of paymentTypes$ | async">
+        <input
+          class="form-check-input"
+          role="radio"
+          type="radio"
+          aria-checked="true"
+          (change)="changeType(type.code)"
+          [value]="type.code"
+          [checked]="type.code == typeSelected"
+        />
+        <label
+          class="cx-delivery-label form-check-label
+                      form-radio-label"
+          for="deliveryMode-{{ type.code }}"
+        >
+          <div class="cx-delivery-mode">{{ type.displayName }}</div>
+        </label>
+      </div>
+    </ng-container>
+
+    <label>
+      <span class="label-content">{{
+        'checkoutType.poNumber' | cxTranslate
+      }}</span>
+      <input #poNumber class="form-control" type="text" />
+    </label>
+
+    <label aria-required="true" *ngIf="typeSelected === 'ACCOUNT'">
+      <span class="label-content required">{{
+        'checkoutType.costCenter' | cxTranslate
+      }}</span>
+      <select class="form-control">
+        <option value=""></option>
+        <option value=""></option>
+      </select>
+    </label>
+  </div>
+</div>
+
+<div class="row cx-checkout-btns">
+  <div class="col-md-12 col-lg-6">
+    <button class="btn btn-block btn-action" (click)="back()">
+      {{ backButtonText | cxTranslate }}
+    </button>
+  </div>
+  <div class="col-md-12 col-lg-6">
+    <button class="btn btn-block btn-primary" (click)="next()">
+      {{ 'common.continue' | cxTranslate }}
```

Comment:
Can we use a constant for this key, like in the button above?

Reply:
This key is used in many places in this way. `backButtonText` is a variable because this button can be "back" or "back to cart" depending on whether this step is the first step.